lib/ansible/plugins/filter/core.py

```diff
@@ -34,7 +34,7 @@
 import uuid
 import yaml
 
-from collections import MutableMapping
+from collections import Mapping, MutableMapping
```

Comment:
Not for this PR but seeing this, I realize that someone will need to audit our code and rename all imports of these base abcs for compatibility with Python-3.8+ https://docs.python.org/3/library/collections.html#module-collections

``` python
from ansible.module_utils.common._compat_collections import Mapping, MutableMapping
```

@webknjaz you wrote _compat_collections (and are watching to see if it makes it into six) so perhaps you're the logical choice to make this change to our codebase?  (Note that if there's too many to do manually, you can probably use the rope refactoring library to make the changes).

Reply:
Okay, I'll take a look.